presto-tests/src/test/resources/resource_groups_config_burst.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
as I mentioned above we don't use `burst` anymore it seems, so we can rename this file with a suffix of `soft_conc_limits` or sth like that.